### ISSUE TITLE
Crash on Odin generated exe when trying to click anything in the function window

### ISSUE THREAD

**Issue by DerTee** [WhiteBox-Systems/whitebox#30]
### What happened?

Actual behaviour:
When choosing an Odin executable to debug and in the function window:
Crash when clicking "Break on function entry" or "Sample whole function" or "{}" or "Sample function input/output..." so basically any button.

Expected behaviour:
Well, not crash, like with C compiled projects.

### System/app details

Exe compiled with Odin `odin version dev-2024-04:aab122ede`

sys_info:
WhiteBox version - 2024-07-16
WhiteBox commit  - 69a84b94717d0812ea9f22a4cd06178202bc1769
Assert level     - 1
Debug info       - false
OS               - Windows 10 Home x64 (10.0.22631)
Service pack     - (0.0)
CPU              - AMD Ryzen 5 2600 Six-Core Processor            
Logical cores    - 12
CPU revision     - 0x802
RAM              - 16.0 GiB
Address space    - 128.0 TiB

### How to reproduce the error

Steps to reproduce:
  1. Compile super simple Hello-World-Type executable with Odin via `odin build . -debug`
  2. D:\apps\whitebox\whitebox_nightly-2024-07-16> whitebox.exe
  3. Menu > Run > Choose executable via button / paste path, makes no difference
  4. In Functions Window click on Button "Break on function entry" of any function -> pause (obviously generating the crash dump)
  5. Crash

Does the error happen consistently with the above steps?
 Always

Minimal reproducible code example:
```odin
package simple
import "core:fmt"

main :: proc() {
    fmt.println("Hellope!")
}
```

### Crash/log attachments & additional info

- [X] crash dump file [whitebox-2024-07-18-22-52-06.dmp.gz](https://github.com/user-attachments/files/16289960/whitebox-2024-07-18-22-52-06.dmp.gz)
- [ ] whitebox-log.txt file

Any additional information/comments:
I get crashes basically anytime I do anything with odin executables, except on rare occasions. Sadly, I can't pinpoint what those rare occasions are. I *think* loading a C project in my editor, doing something there, then switching to debugging an odin executable does something to make things more stable, but I'm not sure. If I find out on my own, I'll comment/edit.

**Comment by azmr** [comment]
Thanks for the report, I'll take a look

**Comment by azmr** [comment]
Fixed in `e6dcf13c` - it was just a minor oversight from a large refactor.
That should be out in a "nightly" today.
